src/hearings/components/multi-level-selector/multi-level-selector.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="multi-select-container" [ngSwitch]="controlType">
+  <div *ngSwitchCase="controlCategory.CHECK_BOX">
+    <div class="govuk-checkboxes__item" *ngFor="let control of multiSelect.controls; let i=index">
+      <div  [formGroup]="control">
+        <input class="govuk-checkboxes__input" id="{{control.value.key}}" 
+          formControlName="selected"
+          name="pos" 
+          type="checkbox">
+        <label class="govuk-label govuk-checkboxes__label" for="{{control.value.key}}">{{control.value.value_en}}</label>
+      </div>
+      <div *ngIf="control.value.child_nodes && control.value.child_nodes.length && control.value.selected" class ="govuk-form-group">
+        <exui-multi-level-selector class="multi-container"
+          [configLevels]="configLevels"
+          [multiSelect]="getValue(control)"
+          [level]="level + 1">
+        </exui-multi-level-selector>
+      </div>
+    </div>
+  </div>
+  <div *ngSwitchCase="controlCategory.TEXT_BOX">
+
+  </div>
+  <div *ngSwitchCase="controlCategory.TEXT_AREA">
+    
+  </div>
```

Comment:
what are these two cases for? seems useless here could they be removed or you support these two types?